README.md

```diff
@@ -17,8 +17,75 @@
 [doi-badge]:           https://zenodo.org/badge/454071954.svg
 [doi-link]:            https://zenodo.org/doi/10.5281/zenodo.10452647
 
-Tools to find evidence for structural variation.
+Tools to gather evidence for structural variation via breakpoint detection.
 
 ## Documentation
 
-Documentation can be found in the [docs folder](docs/01_Introduction.md)
+Documentation can be found in the [docs folder](docs/01_Introduction.md).
+
+## Introduction to the `fgsv` Toolkit
+
+The `fgsv` toolkit contains tools for effective structural variant debugging but are not meant to be used as a structural variant calling toolchain in-and-of-itself.
+Instead, it is better to think of `fgsv` as an effective breakpoint detection and structural variant exploration toolkit.
+
+When describing structural variation, we use the term breakpoint to mean a junction between two loci and the term breakend to refer to one of the loci in a breakpoint.
+Importantly, all point intervals (1-length) reported by this toolkit are 1-based inclusive from the perspective of the reference sequence.
+
+### `fgsv SvPileup`
+
+Collates a pileup of putative structural variant supporting reads.
```

Comment:
```suggestion
Collates pileups of reads over putative SV breakpoints.
```

is this clearer? "putative structural variant supporting" is a mouthful to have between "pileup of" and "reads"